faasmcli/faasmcli/util/env.py

```diff
@@ -59,6 +59,12 @@ def _get_dir(variable, default):
 
 LATEST_CMAKE = "/usr/local/lib/cmake-3.15/bin/cmake"
 
+km_registry_host = environ.get("KM_REGISTRY_HOST")
+if km_registry_host == None:
+    km_registry_host = "localhost"
+KEY_MANAGER_REGISTRY_IP = km_registry_host
+KEY_MANAGER_REGISTRY_PORT = 5000
+WASM_OPT_BIN = 'wasm-opt'
```

Comment:
Can you make the single/ double quotes consistent? We use double quotes for Python strings in this project